robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -1,48 +1,60 @@
+    ...                                  Open contact2 record and create a new relationship with contact 1.
+    ...                                  Verify the parent-child relationship is correctly established.
+
+    [tags]                               W-037650                                             feature:Relationships
+
+    Go To Page                           Details                                              Contact                                               object_id=&{contact2}[Id]
+    Select Tab                           Related
+    Wait Until Loading Is Complete
+    Click Related List Button            Relationships                                        New
+    Wait For Modal                       New                                                  Relationship
+
+    Populate Lookup Field                Related Contact                                      &{contact1}[FirstName] &{contact1}[LastName]
+    Select Value From Dropdown           Type                                                 Parent
+    Click Modal Button                   Save
+    Current Page Should Be               Details                                              Relationship
+    Validate Relation Status Message     &{contact1}[FirstName] &{contact1}[LastName]         &{contact2}[FirstName] &{contact2}[LastName]          Parent
+    Click Link                           link=Show more actions
+    Click Link                           link=Relationships Viewer
+    Wait Until Loading Is Complete
     Capture Page Screenshot
-    Go To Record Home    &{contact1}[Id]
-    Select Tab  Related
-    Load Related List    Relationships
-    ${expected result2}        Catenate    &{contact2}[FirstName] &{contact2}[LastName]    is    &{contact1}[FirstName] &{contact1}[LastName]'s    Child
-    Log to Console    ${expected result2}
-    ${id}    ${status2}    Check Status    &{contact2}[FirstName] &{contact2}[LastName]
-    Should Be Equal As Strings    ${status2}    ${expected result2}
-    Click Related Item Link      Relationships    &{contact2}[FirstName] &{contact2}[LastName]
-    Save Current Record ID For Deletion    npe4__Relationship__c    
+    Go To Page                           Details                                              Contact                                               object_id=&{contact1}[Id]
 
+    Select Tab                           Related
+    Load Related List                    Relationships
 
-*** Keywords ***
+    Current Page Should Be               Details                                              Relationship
+    Validate Relation Status Message     &{contact2}[FirstName] &{contact2}[LastName]         &{contact1}[FirstName] &{contact1}[LastName]          Child
+    ${id}                                Get Current Record Id
+    Store Session Record                 npe4__Relationship__c                                ${id}
```

Comment:
Is there a reason why u r replacing 'Save Current Record ID For Deletion    npe4__Relationship__c ' with this old steps again?

Reply:
This is an accident. Thanks for pointed out. Rectified it.